package.json

```diff
@@ -37,7 +37,7 @@
     "localforage-observable": "2.0.0",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#5a06d1af25ca48927b9efe0d9c42c01c84e7839b",
-    "mattermost-redux": "github:mattermost/mattermost-redux#e3e67f1bc0ebe852d586ee19323fa3e9394a7dd1",
+    "mattermost-redux": "github:mattermost/mattermost-redux#1329ad2223aadc96cf47be6cd2527a553aba3c9e",
```

Comment:
i think you need to change the package-lock too

Reply:
Good point!